src/controllers/facilityController.js

```diff
@@ -26,5 +26,49 @@ const getFacilities=async(req, res)=>{
   }
  }
 
+ /**
+ * PATCH facility
+ * */
+ const updateFacility = async(req, res)=> { 
+    try {    
+      const _facility=await model.Facility.findOne({ where: { id:req.params.id } });
+      if(_facility){
+        const _facilityResult = await model.Facility.update(req.body, { where: { id: req.params.id}});
+        if(_facilityResult) {
+          const data=await model.Facility.findOne({ where: { id:req.params.id } });
+          res.status(200).send({
+            status: "Ok",
+            message: "facility updated succeddfuly",
+            data
+          });
+        }
+      }else{
+        res.status(404).send({
+          status: "bad",
+          message: "facility  not found!",
+          data
+        });
+      }    
+    } catch {
+      res.status(404).send({ message: "facility not found!" });
+    }
+  }
+
+  /**
+ * Delete facility
+ * */
+  const deleteFacility=async(req, res) =>{
+    try {
+      const _facility = await model.Facility.findOne({where:{ id: req.params.id }});
+      if (_facility) {
+        await model.Facility.destroy({where:{ id: req.params.id }});
+        res.status(204).send({message: "facility deleted successfuly"});
```

Comment:
Please be aware that status code `204` means no content therefore, if deleted succeed you will not see any message you will rather see 1... this should be a success code to return this message.